.github/workflows/npm-publish.yml

```diff
@@ -170,8 +165,8 @@ jobs:
               set -e &&
               apk add protobuf-dev &&
               rustup target add aarch64-unknown-linux-musl &&
-              export CC="/aarch64-linux-musl-cross/bin/aarch64-linux-musl-gcc" &&
-              export CXX="/aarch64-linux-musl-cross/bin/aarch64-linux-musl-g++"
+              export CC_aarch64_unknown_linux_musl=aarch64-linux-musl-gcc &&
+              export CXX_aarch64_unknown_linux_musl=aarch64-linux-musl-g++
```

Comment:
`psm` was using the `CC` in a build script, but the build script target is the host, not the final target. Therefore, we need to make sure to only set the cross compilation for the specific host.